### PR TITLE
Docs for nx-p and nx-h* - RSC-634

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-634

### CODE REVIEW

File: gallery/src/styles/NxP/NxPPage.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+const nxPExampleCode = require('./NxPExample.html');
+
+const NxPPage = () => {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <NxP>
+          Using the <NxCode>&lt;p&gt;</NxCode> HTML tag with the
+          <NxCode>.nx-code</NxCode> className applies standard RSC typography, margin and maximum width styles
```
Comment (glenhunter) [inline gallery/src/styles/NxP/NxPPage.tsx:20]: `.nx-code`?

File: gallery/src/styles/NxP/NxPPage.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          for paragraph text. When working in React, the <NxCode>NxP</NxCode> component is available for
```
Comment (glenhunter) [inline gallery/src/styles/NxP/NxPPage.tsx:21]:
```suggestion
          for paragraph text. When working in React, the <NxCode>NxP</NxCode> component is available as
```

**Review by glenhunter — APPROVED** [review]
Aside from a  small error and a text suggestion

File: gallery/src/styles/NxP/NxPPage.tsx (hunk 1 above)
Comment (rpokorny) [reply gallery/src/styles/NxP/NxPPage.tsx]: > .nx-code?

Copy-paste error. 1d51ef11

**Pull request merged**